.github/actions/RunAutomation/UpdateAppBaselines/run.ps1

```diff
@@ -0,0 +1,24 @@
+[Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSReviewUnusedParameter', 'runParameters', Justification = 'The parameter is always passed to the script')]
+param (
+    [Parameter(Mandatory=$true)]
+    $runParameters
+)
+
+# BC Container Helper is needed to fetch the latest artifact version
+Install-Module -Name BcContainerHelper -AllowPrerelease -Force
+Import-Module BcContainerHelper -DisableNameChecking
+Import-Module $PSScriptRoot\..\..\..\..\build\scripts\EnlistmentHelperFunctions.psm1
+
+$newVersion = Update-PackageVersion -PackageName "AppBaselines-BCArtifacts"
+
+$result = @{
+    'Files' = @()
+    'Message' = "No update available"
+}
+
+if ($newVersion) {
+    $result.Files = @("build/Packages.json")
```

Comment:
perhaps consider checking build/Packages.json exists?

Reply:
Extracted functions to get these paths from a single point. If the path doesn't exist, it would have failed way earlier.